storage/tianmu/exporter/data_exporter_txt.cpp

```diff
@@ -92,14 +92,14 @@ void DEforTxt::PutRowEnd() {
     WriteChars(line_terminator_);
 }
 
-size_t DEforTxt::WriteString(const types::BString &str, int len, bool text_or_bin) {
+size_t DEforTxt::WriteString(const types::BString &str, int len, bool text_or_bin, bool isNull) {
   int res_len = 0;
   bool enclose_output{false};
 
   // If you omit the word OPTIONALLY, all fields are enclosed by the ENCLOSED BY character.
   // If you specify OPTIONALLY, the ENCLOSED BY character is used only to enclose values from columns that
   // have a string data type (such as CHAR, BINARY, TEXT, or ENUM):
-  if (str_qualifier_ && (text_or_bin || !opt_enclosed_)) {
+  if (str_qualifier_ && !isNull && (text_or_bin || !opt_enclosed_)) {
```

Comment:
The same problem as above.

Reply:
OK, I will change it to follow Google code style;